tinkergraph-gremlin/src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/process/traversal/step/sideEffect/TinkerGraphStep.java

```diff
@@ -27,26 +34,24 @@
 import org.apache.tinkerpop.gremlin.structure.Edge;
 import org.apache.tinkerpop.gremlin.structure.Element;
 import org.apache.tinkerpop.gremlin.structure.Vertex;
+import org.apache.tinkerpop.gremlin.structure.util.CloseableIterator;
 import org.apache.tinkerpop.gremlin.structure.util.StringFactory;
 import org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerGraph;
+import org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerGraphIterator;
 import org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerHelper;
 import org.apache.tinkerpop.gremlin.util.iterator.IteratorUtils;
 
-import java.util.ArrayList;
```

Comment:
I see a number of changes where the imports are reordered in this fashion. that ordering here is out of step with our style. looking at other classes, we tend to keep the list the `java.*` ones after other libraries. i guess i mostly call your attention to it in case your IDE is automatically making these changes you can adjust the settings so that future PRs don't have these sorts of changes inadvertently mixed in, but i also think it would be good to reverse these sorts of changes in this PR to avoid unnecessary modifications.

Reply:
Thanks. I was about to ask this question in dev email list. We should probably have an intelliJ/eclipse configuration which the developers can import in the IDE and not have to worry about code styling issues.